src/components/molecules/AdminManagementForm.tsx

```diff
@@ -0,0 +1,199 @@
+  const EditMode: any = (props: any) => {
+    return (
+      <ButtonGroup>
+        <Button variant="contained" onClick={() => saveEditMode(props.itemData)}>
+          {t('usersManagement.save')}
+        </Button>
+        <Button variant="contained" onClick={() => cancelEditMode(props.itemData)}>
+          {t('usersManagement.cancel')}
+        </Button>
+      </ButtonGroup>
+    );
+  };
+  const ViewMode = (props: any) => {
+    return (
+      <Button onClick={() => changeEditObject(props.itemData)} variant="contained">
+        {t('usersManagement.edit')}
+      </Button>
+    );
+  };
+  const OrganizationEditList: any = (props: any) => {
+    const orgName = editObject[props.item.id]['organizationValue']
+      ? editObject[props.item.id]['organizationValue']
+      : props.item.roles;
+    const [currentOrganization, setcurrentOrganization] = React.useState(orgName);
+    const orgList = ['walla', 'ynet', 'papaya', 'lahaim', 'or yarok', 'kakal'];
+    const handleChange = (e: any) => {
+      setcurrentOrganization(e.target.value);
+      const newObj = changeCurrentSelectedRoleHelper({ ...editObject }, props.item.id, e.target.value);
+      setEditObject(newObj);
+    };
+    return (
+      <Box>
+        <FormControl fullWidth>
+          <InputLabel id="demo-simple-select-label">Organization</InputLabel>
+          <Select
+            labelId="demo-simple-select-label"
+            name={props.item.id}
+            value={currentOrganization}
+            label="Organization"
+            onChange={handleChange}
+          >
+            {orgList.map((org) => {
+              return (
+                <MenuItem key={org} value={org}>
+                  {org}
+                </MenuItem>
+              );
+            })}
+          </Select>
+        </FormControl>
+      </Box>
+    );
+  };
+  const OrganizationViewMode: any = (item: any) => {
+    if (editObject[item.id]) {
+      return <p>{editObject[item.id]['organizationValue']}</p>;
+    } else {
+      return <p>{item.roles}</p>;
+    }
+  };
+  const cancelEditMode = (el: any) => {
+    const newObj = cancelEditModeHelper(el, { ...editObject });
+    setEditObject(newObj);
+  };
+  const saveEditMode = (element: any) => {
+    const newObj = saveEditModeHelper(element, { ...editObject });
+    setEditObject(newObj);
+  };
+  const changeEditObject = (element: any) => {
+    const newObj = changeEditObjectHelper(element, { ...editObject });
+    setEditObject(newObj);
+  };
+  return (
+    <DialogWithHeader
+      fullWidth
+      maxWidth={'sm'}
+      isShowing={isShowing}
+      title={t('usersManagement.title')}
+      onClose={() => {
+        onClose();
+      }}
+    >
+      <TableContainer style={{ maxHeight: 250 }}>
+        <Table className={classes.table} size="small" aria-label="a dense table">
+          <TableHead>
+            <StyledTableRow>
+              {labels.map((label: string, index: number) => (
+                <StyledTableCell key={index}>
```

Comment:
why don't you expand our `TableView` component so it can be used here?
(basically allow sending elements and not only string/numbers should do it)